lib/tbot/config/config.go

```diff
@@ -677,6 +681,7 @@ func destinationFromURI(uriString string) (bot.Destination, error) {
 // FromCLIConf loads bot config from CLI parameters, potentially loading and
 // merging a configuration file if specified. CheckAndSetDefaults() will
 // be called. Note that CLI flags, if specified, will override file values.
+// TODO: Remove this function.
```

Comment:
(I expect to remove `FromCLIConf` outright, and significantly prune the `CLIConf` struct)

Reply:
This struct is now removed in favor of command-specific structs (and a small dedicated GlobalArgs for the few fields where globals actually make sense)